mapquery.c

```diff
@@ -1284,7 +1285,8 @@ int msQueryByRect(mapObj *map)
       return(MS_SUCCESS);
   }
 
-  msSetError(MS_NOTFOUND, "No matching record(s) found.", "msQueryByRect()");
+  if( lp->debug >= MS_DEBUGLEVEL_V )
```

Comment:
Probably best to put these into braces to avoid bugs in the future:

```
if( lp->debug >= MS_DEBUGLEVEL_V ) {
    msSetError(MS_NOTFOUND, "No matching record(s) found.", "msQueryByRect()");
}
```

Reply:
Agreed! Thanks for the headsup.